src/components/simple-table/SimpleTable.module.scss

```diff
@@ -4,10 +4,10 @@
   width: 100%;
 
   > thead {
-    text-transform: uppercase;
+    text-transform: none;
```

Comment:
What's the reason for this change? In all the designs I have seen the headers are always capitalized.

Reply:
We are slowly moving away from all caps across the board and this was one of the last remaining places in Amino where that text-transform is applied.